packages/blade/src/hooks/useBladeInnerRef.ts

```diff
@@ -23,7 +24,7 @@ const useBladeInnerRef = (
     targetRef,
     (): BladeElementRef => {
       const element = innerRef.current;
-      if (element instanceof HTMLElement) {
+      if (getPlatformType() !== 'react-native' && element instanceof HTMLElement) {
```

Comment:
Just check if on react-native the "focus" method is working as expected by the ref. (Try on the input focus stories)

Reply:
Works. Something else is breaking there in terms of story's styling (we've used alignItem='end' but it is not available on native) but works otherwise